client/extensions/woocommerce/state/ui/orders/test/reducer.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import reducer from '../reducer';
+import { WOOCOMMERCE_UI_ORDERS_SET_PAGE } from 'woocommerce/state/action-types';
+
+describe( 'reducer', () => {
+	describe( 'currentPage', () => {
+		it( 'should have no change by default', () => {
+			const newState = reducer( undefined, {} );
+			expect( newState ).to.eql( {} );
+		} );
+
+		it( 'should store the current page', () => {
+			const action = {
```

Comment:
I will switch my tests to look like this for reducers, this is preferable as it isolates the reducer.